src/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Leaderboard Project</title>
+    <link rel="stylesheet" href="/bootstrap.min.css">
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure that webpack bundles your stylings. Let's kindly try suggestions from the previous reviewer. `npm i`, `npm start` should set up your app completely. I think you'll need to add bootstrap as a dependency or configure it using webpack. Webpack doesn't bundle the minified version of the bootstrap file added.

**Screenshot**
![Capture](https://user-images.githubusercontent.com/63087491/216651715-55d58ea4-686f-42e8-80ff-6abfd122044e.PNG)